main-site/components/testimonials-section/testimonial-card/TestimonialCard.styles.ts

```diff
@@ -47,19 +47,25 @@ const StyledTestimonialCardImage = styled.img<StyledTestimonialCardAuthorYearIma
   }
```

Comment:
Currently, the image is fitting to the container and since the container is rounded, the image will follow suit. Have you tried playing around with the object-fit property?